ui/src/app/pages/analysis/notebook-list.tsx

```diff
@@ -36,23 +50,68 @@ export const NotebookList = withCurrentWorkspace()(class extends React.Component
     notebookList: FileDetail[],
     notebookNameList: string[],
     creating: boolean,
-    loading: boolean
+    loading: boolean,
+    delay: number,
+    cloneLoadingNotebookMsg: boolean;
```

Comment:
I'd prefer a name like "show message" or "should show message" instead of "message" for a boolean